framework/bootstrap.go

```diff
@@ -216,13 +234,18 @@ func (f *framework) watchMeta(linkType string, taskIDs []uint64) {
 			}
 		}
 
+		f.log.Println("Watch ", taskID, " with path ", watchPath)
+
 		// Need to pass in taskID to make it work. Didn't know why.
 		err := etcdutil.WatchMeta(f.etcdClient, taskID, watchPath, stop, responseHandler)
```

Comment:
There's only one place that the watchMeta will block -- it's trying to send event to framework select loop:
```Go
	f.metaChan <- &metaChange{
				from:  taskID,
				who:   linkType,
				epoch: ep,
				meta:  values[1],
			}
```

It only means there are too many events happening. However, it should handle all eventually.